lib/metrics/metrics-tls-cache.c

```diff
@@ -144,14 +144,13 @@ metrics_tls_cache_get_labels_len(void)
 void
 metrics_tls_cache_global_init(void)
```

Comment:
maybe this is a bit too granular for apphook, and we should just have a metrics_global_init()/deinit() functions.

e.g. drop the tls_cache piece from the name. if there will be multiple metrics related global init/deinit chores, we could call all of those from a single metrics_global_init() function.

Reply:
Agree, fixed.